Pages/Navigation/navigation.css

```diff
@@ -0,0 +1,96 @@
+.nav-bar {
+    min-height: 10vh;
+
+    width: 100%;
+    align-items: center;
+    background-color: var(--gray-lighter);
+    font-weight: 600;
+    color: var(--blue-dominant);
+    padding: 1rem;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+}
+
+.list-inline-item {
+    margin-right: 2rem;
+    font-size: 1.8rem;
+}
+
+.nav-section {
+    display: flex;
+    align-items: center;
+}
+
+.nav-section-items {
+    padding: 0 1rem;
+}
+
+.nav-bar-links {
+    display: flex;
+}
+
+
+.nav-icons {
+    display: flex;
+    justify-self: right;
+}
+
+.nav-logo {
+    width: 4rem;
+    height: 4rem;
+    border-radius: 5rem;
+}
+
+.nav-icon-link {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.nav-icon-link .nav-icon-text {
+    font-size: 0.8rem;
+    margin-top: 0.5rem;
+}
+
+.nav-icon-link .nav-icon {
+    color: var(--gray-mild);
+    width: 2rem;
+    height: 2rem;
+    font-size: 2rem;
+}
+
+.search-bar {
+    width: 100%;
+}
+
+.search-bar {
+    background-color: var(--white);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: var(--blue-dominant);
+    border-radius: 5rem;
+    justify-self: right;
+}
+
+.search-bar-input {
+    width: calc(100% - 2rem);
+    border: none;
+    background-color: transparent;
+    padding: 1rem 1rem;
+}
```

Comment:
The outline of search input overflows the parent container on focus. Would suggest to put some border-radius. equal to the search container.

<img width="211" alt="image" src="https://user-images.githubusercontent.com/39650528/153539959-24f1b675-1427-408e-b6d3-5fc1fe8c9a06.png">